examples/guides/services/hospital-service/README.md

```diff
@@ -0,0 +1 @@
+# ballerina-hospital-service
```

Comment:
Let's give a small intro about the purpose of this service, what this service does and how to run this service.